aiidalab_widgets_base/__init__.py

```diff
@@ -13,6 +11,20 @@
 """
 
 
+def load_default_profile():
```

Comment:
Could you move this to the `utils/__init__.py`?

Reply:
This needs to stay here, this code was here before and was just moved to a function.

Reply:
Ah, you are right!